### ISSUE TITLE
NTR: Request for S11 Postflexion larva

### ISSUE THREAD

**Issue by nvsvocabs** [nvs-vocabs/S11#22]
<H2> Required </H2><p><b>Term name (PrefLabel)</b><p>Postflexion larva <p><b>Definition</b><p> Developmental stage from formation of the caudal fin(distal margin of the hypural elements vertical) to attainment of full external meristic complements (fin rays).<p><H2> Optional </H2><p><b>Sources/references</b><p> Ahlstrom, E.H. and O.P. Ball. (1954). Description of eggs and larvae of jack mackerel (Trachurus symmetricus) and distribution and abundance of larvae in 1950 and 1951. Fish. Bull. 56:209-245. || Kendall, Jr. A.W, Ahlstrom, E.H. and Moser, H.G. (1984). Early life history stages of fishes and their characters, American Society of Ichthyologists and Herpetologists, Special publication 1: 11–22. || Mwaluma, J.M., Kaunda-Arara, B., and Strydom, N.A.( 2014). A Guide to Commonly Occurring Larval Stages of Fishes in Kenyan Coastal Waters. WIOMSA Book Series No. 15. xvi. 73pp.<p>

**Comment by danibodc** [comment]
Suggested term:

**Term name (PrefLabel):** postflexion larvae
**Definition:** An ichthyoplankton life stage between the completion of notochord flexion and until the start of metamorphosis. Diagnostic characteristics include formation of the caudal fin and attainment of meristic complements such as fin rays.

**Comment by danibodc** [comment]
We have now created the following term:

S1190 - postflexion larvae
A fish life stage between the completion of notochord flexion and until the start of metamorphosis. Diagnostic characteristics include formation of the caudal fin and attainment of meristic complements such as fin rays.

We chose 'fish' as opposed to 'icythoplankton' in the definition to match FishBase: https://www.fishbase.se/glossary/Glossary.php?q=flexion+larva&sc=is

Note that this will not become live on the NVS until ~0730UT tomorrow.
